### ISSUE TITLE
Problem passing string value as docid

### ISSUE THREAD

**Issue by oktaal** [proycon/python-frog#26]
When creating a Frog instance such as:

```python
from frog import Frog, FrogOptions
frog_instance = Frog(FrogOptions(
    xmlout=True,
    docid='DOCID',
    mwu=False,
    parser=False
))
```

I get the following error:

```
  File "frog_wrapper.pyx", line 80, in frog.FrogOptions.__init__
  File "frog_wrapper.pyx", line 125, in frog.FrogOptions.__setitem__
  File "<stringsource>", line 15, in string.from_py.__pyx_convert_string_from_py_std__in_string
TypeError: expected bytes, str found
```

When I add an `encode()` to `docid='DOCID'` it works:

```python
from frog import Frog, FrogOptions
frog_instance = Frog(FrogOptions(
    xmlout=True,
    docid='DOCID'.encode(),
    mwu=False,
    parser=False
))
```

Reading:

https://github.com/proycon/python-frog/blob/1babaa474b127bc2f6fa5ec4b0ed66be4eb10d89/frog_wrapper.pyx#L124-L137

I think it might need this for `id` and `threads` as well, such as it already does for `skip`, and/or it might need to check if the value has already been encoded to bytes.

**Comment by proycon** [comment]
Thanks, good point indeed! I'll adapt it so it's more intuitive and you can pass strings.

**Comment by proycon** [comment]
Fixed in v0.6.9 release
